install.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# MAKE DIRECTORY SO THAT YOUR HOME DIRECTORY DOESN'T GET DIRTY
+
+mkdir -p junk-drawer
```

Comment:
Might consider using `/tmp` instead. Also, this isn't cleaned up after the script finishes.

Reply:
Oh lol, i didn't use /tmp because some people have a tendency to use a seperate partition for that directory. Also, like i said before, the script can definitely be improved on